src/cryptenroll/cryptenroll-tpm2.c

```diff
@@ -250,7 +250,8 @@ int enroll_tpm2(struct crypt_device *cd,
                 uint32_t pubkey_pcr_mask,
                 const char *signature_path,
                 bool use_pin,
-                const char *pcrlock_path) {
+                const char *pcrlock_path,
+                int *slot_to_wipe) {
```

Comment:
as discussed, please rename to `ret_slot_to_wipe`

and please do `assert(ret_slot_to_wipe)` somewhere next to the other `assert()`s that validate the parameters.